src/lib.rs

```diff
@@ -1,20 +1,25 @@
-// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
```

Comment:
Is there a reason why you changed the copyright?

Reply:
Yeah, "all rights reserved" is contradictory. It was discussed in slack once. I've been slowly changing things over.